tests/driver/runner_test.py

```diff
@@ -2010,7 +2154,40 @@ async def test_accepts_better_cluster_status(self, es):
             "relocating-shards": 0
         }, result)
 
-        es.cluster.health.assert_called_once_with(index=None, params={"wait_for_status": "yellow"})
+        es.cluster.health.assert_called_once_with(params={"wait_for_status": "yellow"})
+
+    @mock.patch("elasticsearch.Elasticsearch")
+    @run_async
+    async def test_cluster_health_with_timeout_and_headers(self, es):
+        es.cluster.health.return_value = as_future({
+            "status": "green",
+            "relocating_shards": 0
+        })
+        r = runner.ClusterHealth()
```

Comment:
we try not to have single value names for variables, lets make this a bit more descriptive